### PR TITLE
chore: clean up up spectral duplicated file

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

Remove it now that we have pinned the sha in mms.

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .spectral.yaml [hunk 1]
```diff
@@ -1,254 +0,0 @@
-# https://meta.stoplight.io/docs/spectral/cb95cf0d26b83-core-functions
-extends:
```
Comment (andreaangiolillo) [inline .spectral.yaml:2]: isn't the spectral in mms still pointing to this file?

Comment (blva) [reply .spectral.yaml:2]: no https://github.com/10gen/mms/blob/48399666b300669ed5717040045e04710a6105f4/scripts/evergreen/openapi/.spectral.yaml#L4

Comment (andreaangiolillo) [reply .spectral.yaml:2]: Thank for checking! another question: is the GH action release updated already to use the new file?

Comment (blva) [reply .spectral.yaml:2]: I'm thinking yes since I updated in the previous PR and Spectral Lint Commited OpenAPI Spec is not failing

Comment (andreaangiolillo) [reply .spectral.yaml:2]: yes, you did  https://github.com/mongodb/openapi/blob/0290379ff307d66aefb30f87209c05495514df1b/.github/workflows/required-spec-validations.yml#L36 👍

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**